relayer-cli/src/utils/relay.ts

```diff
@@ -31,8 +30,7 @@ const getCount = async (veaOutbox: VeaOutboxArbToEth, chainid: number): Promise<
 
 const relay = async (chainid: number, nonce: number) => {
   const routeParams = getBridgeConfig(chainid);
-
-  const veaOutbox = getVeaOutboxArbToEth(routeParams.veaOutbox, process.env.PRIVATE_KEY, routeParams.rpcOutbox);
+  const veaOutbox = getVeaOutbox(routeParams.veaOutbox, process.env.PRIVATE_KEY, routeParams.rpcOutbox, chainid);
```

Comment:
There isn't any. So generally do we validate envs config in the code itself or the environment?

Reply:
IMHO user input (configuration being one) should be validated as early as possible. That allows you to:

- Have more specific constrains in the rest of the application (this will be a string, and not a string OR undefined) which simplifies the process.
- Better UX as you notify the error properly

Reply:
Issue created